src/main/groovy/org/pih/warehouse/inventory/CycleCountStartCommand.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.inventory
+
+import grails.validation.Validateable
+
+class CycleCountStartCommand implements Validateable {
+
+    CycleCountRequest cycleCountRequest
```

Comment:
does the client need to pass the whole request here or just request id and Grails binds the request object to it?

Reply:
we had that discussion recently regarding the `Product`.
Both options are possible, so to provide it like:
```
{
  cycleCountRequest: "UUID"
}
```
or 
```
{
  cycleCountRequest: {
    "id": "UUID"
  }
}
```